### ISSUE TITLE
Hubs stuck on loading screen in Chrome on Android and Windows if webvr/xr flags are enabled

### ISSUE THREAD

**Issue by brianpeiris** [mozilla/hubs#892]
If webxr and/or webvr flags are enabled in `chrome://flags`, hubs will fail to load.

**Comment by brianpeiris** [comment]
There's some odd interaction with webvrpolyfill going on here. aframe.io does not seem to suffer from this problem, so it seems specific to Hubs.

**Comment by johnshaughnessy** [comment]
Burned me for a few hours today. Will be nice to figure this out

**Comment by johnshaughnessy** [comment]
Also fails if `enable-experimental-web-platform-features` is enabled

**Comment by johnshaughnessy** [comment]
I can get past my issue by not loading our service worker:

hub.js:
```
  subscriptions.setRegistrationFailed();
  //if (navigator.serviceWorker) {
  //  try {
  //    navigator.serviceWorker
  //      .register("/hub.service.js")
  //      .then(() => {
  //        navigator.serviceWorker.ready
  //          .then(registration => subscriptions.setRegistration(registration))
  //          .catch(() => subscriptions.setRegistrationFailed());
  //      })
  //      .catch(() => subscriptions.setRegistrationFailed());
  //  } catch (e) {
  //    subscriptions.setRegistrationFailed();
  //  }
  //}
```

**Comment by johnshaughnessy** [comment]
Sorry for the spam -- I withdraw the above comment. It seems to me that it's only the WebXR flags that cause this issue for me, not the WebVR flag.

**Comment by arpu** [comment]
https://github.com/aframevr/aframe/issues/4002

**Comment by brianpeiris** [comment]
It seems the Brave browser is also affected by this. Probably the new Edge as well.

**Comment by brianpeiris** [comment]
Fixed in #1231
